CMP1903_A1_2324/Testing.cs

```diff
@@ -1,19 +1,42 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace CMP1903_A1_2324
 {
-    internal class Testing
+    public class Testing
     {
         /*
          * This class should test the Game and the Die class.
          * Create a Game object, call the methods and compare their output to expected output.
          * Create a Die object and call its method.
          * Use debug.assert() to make the comparisons and tests.
          */
+        public static void Test()
+       {
+           Game game = new Game();
+           int sum_of_dice = game.RollingGame();
+
+            // debug asserting which will allow to find the individual dice rolls
+            Debug.Assert(game.Die1.DiceCurrentValue >= 1 && game.Die1.DiceCurrentValue <= 6, "Dice 1 is out of range");
+            Debug.Assert(game.Die2.DiceCurrentValue >= 1 && game.Die2.DiceCurrentValue <= 6, "Dice 2 is out of range");
+            Debug.Assert(game.Die3.DiceCurrentValue >= 1 && game.Die3.DiceCurrentValue <= 6, "Dice 3 is out of range");
+
+         // the debug asserting tool is used to find the total value and make sure it is the correct number
```

Comment:
The two comments and the code line below need indenting to fit the format. If you are worried about it becoming a block, then add further spacing with further comments.